lms/djangoapps/discussion_api/views.py

```diff
@@ -144,6 +144,9 @@ class ThreadViewSet(_ViewMixin, DeveloperErrorViewMixin, ViewSet):
         * following: If true, retrieve only threads the requesting user is
             following
 
+        * view: "unread" for threads the requesting user has not read, or
+            "unanswered" for question threads with no marked answer.
+
         The topic_id, text_search, and following parameters are mutually
```

Comment:
FYI it is also mentioned [here](https://github.com/edx/edx-platform/pull/8571/files#diff-225c03df42bb7a286124b09118fed6d8L248)